<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CanvasShortcutDialog</class>
 <widget class="QDialog" name="CanvasShortcutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Canvas Shortcut</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="typeGroup">
     <property name="title">
      <string>Type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="typeCombo"/>
      </item>
      <item>
       <widget class="QLabel" name="typeDescription">
        <property name="text">
         <string notr="true">Type description goes here.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="shortcutGroup">
     <property name="title">
      <string>Shortcut</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="widgets::CanvasShortcutEdit" name="shortcutEdit" native="true">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="actionGroup">
     <property name="title">
      <string>Action</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QComboBox" name="actionCombo"/>
      </item>
      <item>
       <widget class="QWidget" name="modifiersWrapper" native="true">
        <layout class="QHBoxLayout" name="modifiersLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="invertedCheckBox">
           <property name="toolTip">
            <string>Inverts the input directions. For example, scrolling up to zoom out instead of zooming in.</string>
           </property>
           <property name="text">
            <string>Inverted</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="swapAxesCheckBox">
           <property name="toolTip">
            <string>Switches horizontal and vertical directions. For example, moving the cursor up and down to change the brush size instead of moving it left and right.</string>
           </property>
           <property name="text">
            <string>Swap Axes</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="constraintsGroup">
     <property name="title">
      <string>Action</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="constraintsCombo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="conflictDescription">
     <property name="text">
      <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Conflict:&lt;/span&gt; conflict description goes here.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>widgets::CanvasShortcutEdit</class>
   <extends>QWidget</extends>
   <header>widgets/canvasshortcutedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>typeCombo</tabstop>
  <tabstop>shortcutEdit</tabstop>
  <tabstop>actionCombo</tabstop>
  <tabstop>invertedCheckBox</tabstop>
  <tabstop>swapAxesCheckBox</tabstop>
  <tabstop>constraintsCombo</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CanvasShortcutDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CanvasShortcutDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
